docs/sources/operators-guide/getting-started-helm-charts/_index.md

```diff
@@ -0,0 +1,298 @@
+     ```
+
+     In this case, your Prometheus server writes metrics to Grafana Mimir that it scrapes from itself.
+
+  1. Start a Prometheus server by using Docker:
+
+     ```bash
+     docker run --network=host -p 9090:9090  -v <path-to>/prometheus.yml:/etc/prometheus/prometheus.yml prom/prometheus
+     ```
+
+## Configure Grafana Agent to write to Grafana Mimir
+
+Make a choice based on whether or not you already have a Grafana Agent set up:
+
+- For an existing Grafana Agent:
+
+  1. Add the following YAML snippet to your Grafana Agent metrics configurations (`metrics.configs`):
+
+     ```yaml
+     remote_write:
+       - url: http://<ingress-host>/api/v1/push
+     ```
+
+     In this case, your Grafana Agent will write metrics to Grafana Mimir, based on what is defined in the existing `metrics.configs.scrape_configs` configuration.
+
+  1. Restart the Grafana Agent.
+
+- For a Grafana Agent that does not exist yet:
+
+  1. Write the following configuration to an `agent.yaml` file:
+
+     ```yaml
+     server:
+       http_listen_port: 12345
+       grpc_listen_port: 54321
+
+     metrics:
+       wal_directory: /tmp/grafana-agent/wal
+
+       configs:
+         - name: agent
+           scrape_configs:
+             - job_name: agent
+               static_configs:
+                 - targets: ["127.0.0.1:12345"]
+           remote_write:
+             - url: http://<ingress-host>/api/v1/push
+     ```
+
+     In this case, your Grafana Agent writes metrics to Grafana Mimir that it scrapes from itself.
+
+  1. Create an empty directory for the write ahead log (WAL) of the Grafana Agent
+
+  1. Start a Grafana Agent by using Docker:
+
+     ```bash
+     docker run --network=host  -v <path-to-wal-directory>:/etc/agent/data -v <path-to>/agent.yaml:/etc/agent/agent.yaml grafana/agent
+     ```
+
+## Query data in Grafana
+
+First install Grafana, and then add Mimir as a Prometheus data source.
+
+1. Start Grafana by using Docker:
+
+   ```bash
+   docker run --rm --name=grafana --network=host grafana/grafana
+   ```
+
+1. In a browser, go to the Grafana server at [http://localhost:3000](http://localhost:3000).
+1. Sign in using the default username `admin` and password `admin`.
+1. On the left-hand side, go to **Configuration** > **Data sources**.
+1. Configure a new Prometheus data source to query the local Grafana Mimir server, by using the following settings:
+
+   | Field | Value                              |
+   | ----- | ---------------------------------- |
+   | Name  | Mimir                              |
+   | URL   | http://\<ingress-host\>/prometheus |
+
+   To add a data source, see [Add a data source](https://grafana.com/docs/grafana/latest/datasources/add-a-data-source/).
+
+1. Verify success:
+
+   You should be able to query metrics in [Grafana Explore](https://grafana.com/docs/grafana/latest/explore/),
+   as well as create dashboard panels by using your newly configured `Mimir` data source.
```

Comment:
I would consider linking the "Monitoring Grafana Mimir" documentation page to install Mimir dashboards.

Reply:
done